queenbee/schema/operator.py

```diff
@@ -0,0 +1,135 @@
+"""Queenbee Task Operator class.
+
+A task operator includes the information for executing tasks locally or in a container.
+"""
+from queenbee.schema.qutil import BaseModel
+from pydantic import Schema, validator
+from typing import List, Any, Set
+from enum import Enum
+
+
+class Language(BaseModel):
+    """Required programming language."""
+    name: str = Schema(
+        'python',
+        description='Language name'
+    )
+    
+    version: str = Schema(
+        None,
+        description='Language version requirements. For instance ==3.7 or >=3.6'
+    )
+    
+    _valid_languages: Set[str] = set(['python', 'nodejs', 'bash'])
+
+    # NOTE: yaml convertion doesn't play well with Enum. Use validators like this
+    # instead.
+    @validator('name')
+    def check_language(cls, v):
+        assert v.lower() in cls._valid_languages, \
+            '{} is not a valid language. Try one of these languages:\n - {}'.format(
+                v, '\n - '.join(cls._valid_languages)
+            )
+        return v.lower()
+
+
+# TODO: This will most likely change. Too much to take on!
+class App(BaseModel):
+    """Local application."""
+    name: str = Schema(..., description='App name')
+    version: str = Schema(
```

Comment:
PEP8 👮 says that you should have one blank line. Same goes for the `Package` class.

Reply:
I have made everyone a PEP8 👮‍♂️! So proud of myself for this one. 😂 Thanks.